build.gradle

```diff
@@ -146,6 +153,24 @@ def convertGitBranch = { gitBranch ->
     }
 }
 
+createExe {
+    dontWrapJar = true
+    stayAlive = true
+    version = "${versionMajor}.${versionMinor}"
+    textVersion = displayVersion
+    companyName = orgName
+    jreMinVersion = "${targetCompatibility}.0" // launch4j requires it's in the form x.x.x, thus this currently renders to 1.8.0
```

Comment:
Oh, and it may be because of Dialogs: http://code.makery.ch/blog/javafx-dialogs-official/ (not 100% sure, but there was a reason for this requirement).